testing-daml-models/daml-scripts/1_first_script.md

```diff
@@ -0,0 +1,58 @@
+The first thing you need to do when writing a DAML script is to include the DAML script package in
+your dependencies. 
+
+- First, **open the IDE and wait for it to load**
+- After that, click on `daml.yaml`{{open}} to open the `daml.yaml` file.
+- Add the line
+
+  <pre class="file" data-target="clipboard">
+    - daml-script
+  </pre>
+
+  under the `dependencies` stanza. 
+
+Now open the `daml/User.daml`{{open}} file and just before the module declaration line add the
+language pragma
+
+<pre class="file" data-target="clipboard">
+{-# LANGUAGE ApplicativeDo #-}
+</pre>
+
+and just after the module declaration add the `Daml.Script` library to the imports. Your module
```

Comment:
Maybe add the explanation for the pragma directly here. I head to jump up and down to figure out the explanation.
```suggestion
and just after the module declaration add the `Daml.Script` library to the imports. The `ApplicativeDo` language pragma is a trick to be able to reuse the `do` notation you already know from DAML scenarios and upgrade blocks in choices. Your module
```

Reply:
That's true, I think it's better to leave it in, otherwise, people will run into a tricky situation when they run into an error they won't understand.

Reply:
moved the explanations up.